tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -907,6 +907,33 @@ def _get_engine_dtypes_from_node(node, key):
   return [dtypes._TYPE_TO_STRING[dtype] for dtype in node.attr[key].list.type]
 
 
+def _construct_function_from_graph_def(func, graph_def, frozen_func=None):
+  """Rebuild function from graph_def."""
+  if frozen_func is None:
+    frozen_func = func
+  rebuilt_func = wrap_function.function_from_graph_def(
+      graph_def, [tensor.name for tensor in frozen_func.inputs],
+      [tensor.name for tensor in frozen_func.outputs])
+  rebuilt_func.graph.structured_outputs = nest.pack_sequence_as(
+      func.graph.structured_outputs, rebuilt_func.graph.structured_outputs)
+  # Copy structured input signature from original function (used during
+  # serialization)
+  rebuilt_func.graph.structured_input_signature = (
+      func.structured_input_signature)
+  return rebuilt_func
+
+
+def _apply_inlining(func):
+  """Apply an inlining optimization to the function's graph definition."""
+  lower_control_flow = True
+  aggressive_inlining = False
+
+  graph_def = convert_to_constants._run_inline_graph_optimization(
```

Comment:
_run_inline_graph_optimization is a protected member and this can't pass lint check.